demos/kubeflow/README.md

```diff
@@ -72,7 +86,9 @@ If the values are not defined, apply a new user/password with the juju commands.
 
 ```sh
 $ juju config minio access-key=minio
-$ juju config minio secret-key=minio123
+$ kubectl get secret -n kubeflow minio-secret -o yaml | grep MINIO_SECRET_KEY | awk '{ print $2}' | base64 -d | xargs -I {} juju config minio secret-key={}
```

Comment:
Just a comment: can you make sure that MINIO_SECRET_KEY is available on the VM itself? I know It is available in the jupyter notebook with some check boxes checked, but not sure it's available on VM itself

Reply:
Not very sure about the question here. Which VM you mention here?

Reply:
nvm, I should read the whole command.